packages/destination-actions/src/destinations/stackadapt-audiences/index.ts

```diff
@@ -61,7 +133,45 @@ const destination: DestinationDefinition<Settings> = {
   },
   actions: {
     forwardProfile,
-    forwardAudienceEvent
+    forwardAudienceEvent,
+    onDelete: {
```

Comment:
I'd suggest refactor the onDelete to a separate folder like `forwardProfile` and `forwardAudienceEvent`